src/css/globals.css

```diff
@@ -0,0 +1,172 @@
+/* body and html styles */
+body {
+    display: flex;
+    flex-direction: column;
+    overflow-x: hidden;
+  }
+
+/* Page styles */
+.page__header {
```

Comment:
wondering if its possible to isolates these styles near the header code instead of making them global?